scss/_variables.scss

```diff
@@ -459,20 +489,30 @@ $position-values: (
 //
 // Settings for the `<body>` element.
 
-$body-bg:                   $white !default;
-$body-color:                $black !default;
 $body-text-align:           null !default;
+$body-color:                $black !default; // Boosted mod: instead of `$gray-900`
+$body-bg:                   $white !default;
+
+$body-emphasis-color:       $black !default;
+
+$body-secondary-color:      $gray-700 !default; // Boosted mod: instead of `rgba($body-color, .75)`
+$body-secondary-bg:         $gray-300 !default; // Boosted mod: instead of `$gray-200`
+
+$body-tertiary-color:       rgba($body-color, .5) !default;
+$body-tertiary-bg:          $gray-100 !default;
```

Comment:
IMO, it should be `$gray-200` to see something ?

Reply:
TBH right now IDK. I'm still waiting for the list of grays we can use in ODS.